src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -129,9 +133,15 @@ private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationTo
                         project.InstalledVersion = installedVersion.Identity.Version;
                         hash.Add(installedVersion.Identity.Version);
                         project.AutoReferenced = installedVersion.IsAutoReferenced;
+
+                        if (project.NuGetProject.ProjectStyle.Equals(ProjectStyle.PackageReference))
```

Comment:
How about `if (IsRequestedVisible)` ? 

```suggestion
                        if (IsRequestedVisible)
```

Reply:
Can't some projects be PR and others be PC? I thought this was project-by-project determination

Reply:
You are right, it was a different method from the above one.
I'm pretty sure I provided the same feedback to Martin :) 

Resolving this.